src/App.tsx

```
@@ -1,33 +1,76 @@
+								]}
+								rehypePlugins={[rehypeKatex]}
+								components={{
+									code(props) {
+										const { children, className } = props;
+										const match = /language-(\w+)/.exec(
+											className || ""
+										);
+										return match ? (
+											<SyntaxHighlighter
+												PreTag="div"
+												children={String(
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)